static/js/containers/App_test.js

```diff
@@ -127,4 +128,20 @@ describe("App", () => {
       )
     })
   })
+  ;[true, false].forEach(editingWidgets => {
+    it(`${
+      editingWidgets ? "shows" : "doesn't show"
```

Comment:
Take or leave: you can use `shouldIf()` here. 

`${shouldIf(editingWidgets)} show an overlay disabling parts of the UI`